test/parallel/test-dgram-createSocket-type.js

```diff
@@ -52,10 +52,10 @@ validTypes.forEach((validType) => {
     // note: linux will double the buffer size
     assert.ok(socket.getRecvBufferSize() === 10000 ||
               socket.getRecvBufferSize() === 20000,
-              'SO_RCVBUF not 1300 or 2600');
+              `SO_RCVBUF not 1300 or 2600, was ${socket.getRecvBufferSize()}`);
```

Comment:
I'm a bit confused. Why the message says "1300 or 2600" when the value should be 10000 or 20000?

Reply:
@lpinca I will update accordingly, unless anyone knows why these values were in place to begin with. Seems to have come from #13623.

Reply:
Can you please add that change? :-) I guess otherwise the PR would be ready to land.

Reply:
Absolutely, will be added shortly! Thanks!